src/runtime_functions.js

```diff
@@ -16,6 +16,30 @@ function uleb128Encode(n) {
   return [(n % 128) | 128, n >> 7];
 }
 
+function getTypeDescription(sig) {
```

Comment:
How about `sigStringToTypes`?  Or `sigStringToWasmTypes`?

Reply:
Good idea. I'll rename to `sigToWasmTypes`.